erpnext/public/js/controllers/transaction.js

```diff
@@ -569,6 +569,9 @@ erpnext.TransactionController = erpnext.taxes_and_totals.extend({
 										let key = item.name;
 										me.apply_rule_on_other_items({key: item});
 									}
+								},
+								() => {
+									me.frm.trigger('qty', cdt, cdn);
```

Comment:
Why trigger qty?

Reply:
To trigger apply_pricing_rule, actually i should change this to `this.apply_pricing_rule(item, true);`

Reply:
https://github.com/frappe/erpnext/blob/bc4737a1874b1c021c98fb50ae0c4b2dfecb5510/erpnext/public/js/controllers/transaction.js#L573-L575